langcodes/__init__.py

```diff
@@ -142,6 +142,11 @@ def get(tag: str, normalize=True) -> 'Language':
         >>> Language.get('und')
         Language.make()
 
+        This function is idempotent, in case you already have a language tag:
```

Comment:
Should this be "in case you already have a language object"?

Reply:
Ah, right.